sandbox/src/mocks/stu3/changeShortlistAndSendForTriage/responses/MinimalRequest.json

```diff
@@ -198,8 +251,69 @@
                                     }
                                 },
                                 "status": "accepted"
+                            },
+                            {
+                                "type": {
+                                  "coding": [
+                                    {
+                                      "system": "http://hl7.org/fhir/ValueSet/encounter-participant-type",
+                                      "code": "CON"
+                                    }
+                                  ]
+                                },
+                                "actor": {
+                                  "identifier": {
+                                    "system": "http://fhir.nhs.net/Id/sds-user-id",
+                                    "value": "021600556514"
+                                  }
+                                },
+                                "status": "accepted"
```

Comment:
How do I get this to appear in the response?

Reply:
I was going to add this to the request itself so the response could display the fields, but given this endpoint is now deprecated, I opted for reverting the changes back.